test/test_enzyme.jl

```diff
@@ -16,10 +16,8 @@ const maximum_diffusivity = 100
 Change diffusivity of model to `diffusivity`.
 """
 function set_diffusivity!(model, diffusivity)
-    closure = VerticalScalarDiffusivity(; κ=diffusivity)
-    names = tuple(:c) # tracernames(model.tracers)
-    closure = with_tracers(names, closure)
-    model.closure = closure
+    closure = model.closure
+    fill!(closure.κ.c, diffusivity)
```

Comment:
Can you change it back? We wanted this specific test (a parameter estimation-type test)

Reply:
see comment below